nss-day-cohort-47/reactive-nutshell-three-https-in-a-pod
# Em starting public chat

# Description
added a public chat feature consisting of a message board and an input field. once a message is typed and sent, it should appear in the message board. each message also has a delete button to remove unwanted messages.

Fixes # (issue)
## Type of change
Please delete options that are not relevant.


# Testing Instructions
sign into the app and click on the messages tab. Type in a message and send it to see it appear on the board. deleting a message is also an option.

# Checklist:
